examples/vmi-ephemeral.yaml

```diff
@@ -20,5 +20,5 @@ spec:
   terminationGracePeriodSeconds: 0
   volumes:
   - containerDisk:
-      image: registry:5000/kubevirt/cirros-container-disk-demo:devel
+      image: kubevirt/cirros-container-disk-demo:latest
```

Comment:
Please remove unrelated changes from the PR

Reply:
@danielBelenky Leaked by mistake. Done